src/inabox/inabox-viewport.js

```diff
@@ -262,35 +262,23 @@ export class ViewportBindingInabox {
   }
 
   /** @override */
-  getLayoutRectAsync(el) {
+  getGlobalClientRect() {
```

Comment:
In this case, I'd rather call it `getRootClientRect` or such.

Reply:
Yeah. `getRootClientRect` is better name. Also, let's decide do we call them all `Async` or none.

Reply:
Changed to `getRootClientRectAsync`. I like `Async` because it clearly indicate that this is an async function, that's different from `getBoundingClientRect()`